content/en/tracing/trace_collection/custom_instrumentation/ios/otel.md

```diff
@@ -0,0 +1,341 @@
+---
+title: iOS and tvOS applications custom instrumentation using OpenTelemetry API
+kind: documentation
+code_lang: swift
+type: multi-code-lang
+code_lang_weight: 80
+description: 'Instrument your iOS and tvOS application with OpenTelemetry API, to send traces to Datadog.'
+aliases:
+- /tracing/trace_collection/otel_instrumentation/ios/
+- /tracing/trace_collection/custom_instrumentation/otel_instrumentation/ios
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+    - link: '/opentelemetry/guide/otel_api_tracing_interoperability'
+      tag: 'Documentation'
+      text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
+---
+
+{{% otel-custom-instrumentation %}}
+
+## Requirements and limitations
+
+<!-- TODO: version needs to corrected after release -->
+- DatadogTrace for iOS and tvOS version 1.0.0+.
+
+The following OpenTelemetry Swift features are implemented in the Datadog library as noted:
+
+| Feature | Support notes |
+|-|-|
+| [Span attributes][2] | Supported |
+| [Span events][3] | Unsupported |
+| [Span exporters][5] | Unsupported |
+| [Span links][1] | Supported |
+| [Span processors][4] | Unsupported |
+| Context propagation | [W3C Trace Context and Datadog header formats][6] are enabled by default.  |
+
+## Tracing iOS Applications with OpenTelemetry
+
+1. Declare the library as a dependency depending on your package manager:
+
+{{< tabs >}}
+{{% tab "CocoaPods" %}}
+
+You can use [CocoaPods][7] to install `dd-sdk-ios`:
+```
+pod 'DatadogCore'
+pod 'DatadogTrace'
+```
+
+{{% /tab %}}
+{{% tab "Swift Package Manager (SPM)" %}}
+
+To integrate using Apple's Swift Package Manager, add the following as a dependency to your `Package.swift`:
+```swift
+.package(url: "https://github.com/Datadog/dd-sdk-ios.git", .upToNextMajor(from: "2.0.0"))
+```
+
+In your project, link the following libraries:
+```
+DatadogCore
+DatadogTrace
+```
+
+{{% /tab %}}
+{{% tab "Carthage" %}}
+
+You can use [Carthage][8] to install `dd-sdk-ios`:
+```
+github "DataDog/dd-sdk-ios"
+```
+
+In Xcode, link the following frameworks:
+```
+OpenTelemetryApi.xcframework
+DatadogInternal.xcframework
+DatadogCore.xcframework
+DatadogTrace.xcframework
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Initialize the library with your application context and your [Datadog client token][9]. For security reasons, you must use a client token: you cannot use [Datadog API keys][10] to configure the `dd-sdk-ios` library as they would be exposed client-side in the iOS application IPA byte code.
+
+For more information about setting up a client token, see the [client token documentation][11].
+
+{{< site-region region="us" >}}
+{{< tabs >}}
+{{% tab "Swift" %}}
```

Comment:
It looks like we can remove all of the Swift tabs since it's the only one.

Reply:
all iOS platforms support Swift and objective-c and we want to make sure that from the documentation it is clear that support is only for Swift.

Reply:
Gotcha, that makes sense. I think the format of using a tab for this could be a bit confusing though. I think we could remove these tabs and note that only Swift is supported in the _Requirements and limitations_ section or in an alert banner. What do you think of that?

Reply:
that would work, I just made the suggested change.

Reply:
Sounds good! Let me know once the change is pushed up, and I can review/approve. 😄